src/device-api/v1.ts

```diff
@@ -94,29 +95,21 @@ const handleLegacyServiceAction = (action: CompositionStepAction) => {
 router.post('/v1/apps/:appId/stop', handleLegacyServiceAction('stop'));
 router.post('/v1/apps/:appId/start', handleLegacyServiceAction('start'));
 
-const rebootOrShutdown = async (
-	req: express.Request,
-	res: express.Response,
-	action: deviceState.DeviceStateStepTarget,
-) => {
-	const override = await config.get('lockOverride');
-	const force = checkTruthy(req.body.force) || override;
-	try {
-		const response = await deviceState.executeStepAction({ action }, { force });
-		res.status(202).json(response);
-	} catch (e: any) {
-		const status = e instanceof UpdatesLockedError ? 423 : 500;
-		res.status(status).json({
-			Data: '',
-			Error: (e != null ? e.message : undefined) || e || 'Unknown error',
-		});
-	}
+const handleDeviceAction = (action: deviceState.DeviceStateStepTarget) => {
+	return async (req: AuthorizedRequest, res: Response, next: NextFunction) => {
+		const force = checkTruthy(req.body.force);
+		try {
+			await actions.executeDeviceAction({ action }, force);
+			// QUESTION: this used to return { Data: 'OK', Error: null }, okay to change to just 'OK'?
+			return res.status(202).send('OK');
```

Comment:
👀

Reply:
Answering the question. I'd argue it's not OK to change the interface. I would think it's only OK for error responses, since errors are generally not expected to contain any relevant data. For 2xx responses, we need to maintain the interface until we can go to a major version

Reply:
Reverted in 3b20224d149cfb0ae19407bd422dcf8c86f45fb3